tests/Neo.Plugins.RpcServer.Tests/UT_RpcServer.cs

```diff
@@ -39,7 +40,28 @@ public void TestSetup()
             _memoryStore = new MemoryStore();
             _memoryStoreProvider = new TestMemoryStoreProvider(_memoryStore);
             _neoSystem = new NeoSystem(TestProtocolSettings.SoleNode, _memoryStoreProvider);
-            _rpcServer = new RpcServer(_neoSystem, RpcServerSettings.Default);
+            _rpcServer = new RpcServer(_neoSystem, new RpcServerSettings
```

Comment:
why change this?

Reply:
I need RpcServer session enabled to traverse iterator

Reply:
I think this can work.
```csharp
_rpcServer = new RpcServer(_neoSystem, RpcServerSettings.Default with { SessionEnabled = true });
```